### DISCUSSION TITLE
Customizable method definitions

### DISCUSSION THREAD

**Discussion by aeharding** [mdn/mdn-community#336] [MDN Plus]
It would be cool if MDN Plus subscribers could customize how methods are documented. For example, `document.querySelector()` vs `Document.prototype.querySelector()`

https://elk.harding.dev/social.harding.dev/@alex/109908979997142432

Obviously the most approachable option should be chosen for non-plus users, but it would be really cool if this could be customized for power users (plus subscribers).

**Comment by pransh15** [comment]
That's an interesting take. The Plus team to considering working on this, thanks @aeharding! 🙌
